docker-compose.debug.yml

```diff
@@ -0,0 +1,24 @@
+version: '3.4'
+
+services:
```

Comment:
docker-compose.debug.yml ? Why it's called debug ?

Reply:
It was auto-generated by VS Code. I have change debug -> dev